brave-instrumentation/aws-java-sdk-v2-core/src/main/java/brave/instrumentation/awsv2/TracingClientOverrideConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016-2018 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.instrumentation.awsv2;
+
+import brave.http.HttpTracing;
+import software.amazon.awssdk.core.client.config.ClientOverrideConfiguration;
+
+public class TracingClientOverrideConfiguration {
```

Comment:
only request is to make this component (rename) AwsSdkTracing or AwsTracing (so we can configure other things later as needed). Similar to [KafkaTracing](https://github.com/openzipkin/brave/blob/master/instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaTracing.java). Then make the type final and add a package private ctor.

Then, you could either do the build() mentioned here or expose a factory method 
`executionInterceptor()` instead